src/jit/inline.h

```diff
@@ -447,10 +447,10 @@ class InlineResult
     // Report/log/dump decision as appropriate
     void Report();
 
-    Compiler*               m_Compiler;
+    Compiler*               m_Compiler;   // root compiler instance
     InlinePolicy*           m_Policy;
```

Comment:
I wonder if this (and all other related cases in this class, such as the constructor) should rename it m_RootCompiler to make this clear. Your call.